Cargo.toml

```diff
@@ -54,7 +54,7 @@ futures-util = "0.3.25"
 fvm_ipld_blockstore = "0.1.1"
 fvm_ipld_car = "0.6.0"
 fvm_ipld_encoding = "0.3.0"
-graphsync = { git = "https://github.com/retrieval-markets-lab/rs-graphsync.git", rev = "0a53cf5" }
+graphsync = { git = "https://github.com/kckeiks/rs-graphsync.git",  branch = "downgrade-cid" }
```

Comment:
I was in dependency hell for a while because of `cid` and `libipld` dependencies. I ended up having to fork `rs-graphsync` and downgrade `libipld`. Opened #239 to address it in the future.